nomad/data_source_acl_policies_test.go

```diff
@@ -0,0 +1,88 @@
+package nomad
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/nomad/api"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccDataSourceNomadAclPolicies_Basic(t *testing.T) {
+	dataSourceName := "data.nomad_acl_policies.test"
+	numPolicies := 2
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testProviders,
+		Steps: []resource.TestStep{
+			{
+				PreConfig:   testAccCreateNomadAclPolicies(t, numPolicies),
```

Comment:
work-around for creating acl policies; when done in a terraform config, the test resulted in a non-empty plan after refresh such that the individual ACL policy resources were getting deleted and the ACL policies data-source needed to update after refresh

Reply:
neat! i think i need to look at this for some of our other tricky testing scenarios.